work-management/README.md

```diff
@@ -0,0 +1,15 @@
+# Work Management
+We aim to track all work at Tilt in [Clubhouse](https://app.clubhouse.io/windmill) and Google docs, together as the single source of truth, system of record. In particular, this includes future plans and discussions, prioritization and existing work, and past work. This includes product changes, as well as other typical activities requiring collaboration (e.g. planning for a conference).
```

Comment:
What are some things we track in google docs as a source of truth? I thought we were trying to move everything to CH at this point.

Reply:
@maiamcc : I'd love to have a future where everything is in Clubhouse. The one thing I can think of that we _do_ have in Google docs now is our project charters. We could potentially just use Clubhouse epics/milestones to manage that content. I'd be open to do that. But I was thinking more from the perspective of this PR, I wanted to reflect as close as possible our current workflow, so that it wouldn't be burdensome for people to try to follow any new process immediately. So for example, in a future PR, we could remove Google docs here and detail out how we do charters in Clubhouse epics/milestones.

Clubhouse has a google docs competitor (more like Notion competitor) that is pretty decent and is pretty much integrated into the UI of Clubhouse. Unfortunately [right now it's beta](https://clubhouse.io/blog/write-beta/) and hard to get access to I think. Just wanted to mention this as a future optimization where if we wanted more doc-style tracking of work, Clubhouse will offer it.

Reply:
Oh right--yeah that makes sense.